server/.env.example

```
@@ -0,0 +1,5 @@
+DATABASE_URL=sqlite:///./sky_casa_web.db
+JWT_SECRET=change-me
+JWT_EXPIRE_MINUTES=4320
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The JWT_EXPIRE_MINUTES key should go before the JWT_SECRET key

(UnorderedKey)